upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -300,6 +300,17 @@ func (c *ApplyClusterCmd) Run() error {
 		}
 	}
 
+	var nodeDockerConfig string
+	{
+		secret, _ := secretStore.Secret("nodedockercfg")
+		if secret != nil {
+			nodeDockerConfig, err = secret.AsString()
+			if err != nil {
+				return fmt.Errorf("error node docker config not a string? %q: %v", nodeDockerConfig, err)
```

Comment:
I think nodeDockerConfig is going be nil / "".  So I don't think we print it, and I think that gets rid of `nodeDockerConfig` entirely.

Also, can we have a comment here - like `// Verify that dockercfg is a string`